src/core/processors/processorfactory.cpp

```diff
@@ -74,10 +74,26 @@ bool ProcessorFactory::registerObject(ProcessorFactoryObject* processor) {
     return true;
 }
 
-std::unique_ptr<Processor> ProcessorFactory::create(std::string_view key) const {
-    return Parent::create(key, app_);
+std::unique_ptr<Processor> ProcessorFactory::create(std::string_view, InviwoApplication*) const {
```

Comment:
Is this just for debugging?
Function could be removed?

Reply:
We need to implement this since it is part of the factory interface